docs/content.zh/docs/deployment/resource-providers/yarn.md

```diff
@@ -237,7 +237,14 @@ The configuration parameter for specifying the REST endpoint port is [rest.bind-
 
 ### User jars & Classpath
 
-By default Flink will include the user jars into the system classpath when running a single job. This behavior can be controlled with the [yarn.per-job-cluster.include-user-jar]({{< ref "docs/deployment/config" >}}#yarn-per-job-cluster-include-user-jar) parameter.
+**Session Mode & Application Mode**
+
+When deploying Flink with Session/Application Mode on Yarn, only the JAR file specified in startup command will be recognized as user-jars and included into user classpath.
```

Comment:
For YARN application mode, the JAR file specified in startup command will be added to system classpath by default. It could also be controlled by `yarn.per-job-cluster.include-user-jar` although the name is misleading.